### PR TITLE
allow value of GRUB_TERMINAL to be empty

### PR DESCRIPTION
**Pull request by leifliddy**
Fixes # 
https://github.com/OSInside/kiwi/issues/2261

Changes proposed in this pull request:
Allow value of GRUB_TERMINAL to be an empty string
(As of this moment) setting GRUB_TERMINAL to console causes garbage rendering on Apple Silicon systems. So GRUB_TERMINAL needs to contain a blank value (or not be set at all) if you want a properly rendered grub boot menu on Apple Silicon systems.

### CODE REVIEW

File: kiwi/schema/kiwi.rnc [hunk 1]
```diff
@@ -37,7 +37,7 @@ vhd-tag-type = xsd:token {pattern = "[0-9a-f]{8}-([0-9a-f]{4}-){3}[0-9a-f]{12}"}
 groups-list = xsd:token {pattern = "[a-zA-Z0-9_\-\.:]+(,[a-zA-Z0-9_\-\.:]+)*"}
 arch-name = xsd:token {pattern = "(x86_64|i586|i686|ix86|aarch64|arm64|armv5el|armv5tel|armv6hl|armv6l|armv7hl|armv7l|ppc|ppc64|ppc64le|s390|s390x|riscv64)(,(x86_64|i586|i686|ix86|aarch64|arm64|armv5el|armv5tel|armv6hl|armv6l|armv7hl|armv7l|ppc|ppc64|ppc64le|s390|s390x|riscv64))*"}
 portnum-type = xsd:token {pattern = "(\d+|\d+/(udp|tcp))"}
-grub_console = xsd:token {pattern = "(console|gfxterm|serial)( (console|gfxterm|serial))*"}
+grub_console = xsd:token {pattern = "(\s*|console|gfxterm|serial)( (console|gfxterm|serial))*"}
```
Comment (schaefi) [inline kiwi/schema/kiwi.rnc:40]: Could we make this more explicit by providing a ```none``` console type ? Something like this

```
grub_console = xsd:token {pattern = "(none|console|gfxterm|serial)( (console|gfxterm|serial))*"}
```

It imho makes it more clear what is expected to happen with the console setting and could also be
found better in the documentation. 

Talking about the docs, could you add some words about this in
```doc/source/image_description/elements.rst```

Thanks much

**Review by schaefi — CHANGES_REQUESTED** [review]
Thanks, I'm good with the change. I would just propose that we come up with a name for setting no console and some doc hint about it.

File: kiwi/schema/kiwi.rnc (hunk 1 above)
Comment (schaefi) [reply kiwi/schema/kiwi.rnc:40]: @leifliddy any thoughts ? Thanks

Comment (leifliddy) [reply kiwi/schema/kiwi.rnc:40]: Yup, that makes perfect sense. Sorry, I was on a motorcycle trip over the last several days. I'll try and make these changes tonight.

Comment (schaefi) [reply kiwi/schema/kiwi.rnc:40]: excellent thanks much :+1:

**Review by schaefi — APPROVED** [review]
looks good to me :+1: Thanks

**Pull request merged**